GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Logging;
+
+namespace GitFyle.Core.Api.Brokers.Loggings
+{
+    public class LoggingBroker : ILoggingBroker
+    {
+        private readonly ILogger logger;
+
+        public LoggingBroker(ILogger logger) => this.logger = logger;   
```

Comment:
```suggestion
        public LoggingBroker(ILogger logger) => 
            this.logger = logger;   
```

Reply:
This has been updated. Please review to confirm issue has been resolved.